src/desktop/apps/experimental-app-shell/server.tsx

```diff
@@ -92,6 +95,14 @@ app.get(
           styleTags,
           pageType,
 
+          // TODO: Follow up with Justin on stripe include in scripts.jade
+          // template. Right now the order app explicitly sets this to true,
+          // but looking over there it seems we're already including it on
+          // every page, with `async` and `defer`.
+          // options: {
+          //   stripev3: true, // for order app
```

Comment:
@zephraph / @sweir27 - thoughts on this? Looking at the [template here](https://github.com/artsy/force/blob/master/src/desktop/components/main_layout/templates/scripts.jade#L121-L125) we're already including stripe asynchronously without explicitly setting this local. Any reason why this is explicitly set to true, thereby removing the asynchrony?

Reply:
For certain pages in the order app (namely, the payment page), we need Stripe to be present as soon as possible when the page loads (otherwise the credit card form will be unresponsive and the user won't be able to check out).

Reply:
I think now that this is in the global router shell and loads up front it should be ok, we can play with this setting. (Doesn't affect anything when running with `EXPERIMENTAL_APP_SHELL=false`)